workers/directorIndexWorker.js

```diff
@@ -0,0 +1,141 @@
+var request = require('request');
+
+var initialQuery = "https://search-moviebot-squiezr3n3t55xzc3c46awndia.us-east-1.es.amazonaws.com/movies-v8/movie/_search?scroll=5m";
+var scrollQuery = "https://search-moviebot-squiezr3n3t55xzc3c46awndia.us-east-1.es.amazonaws.com/_search/scroll";
+var insertDirectorName = "https://search-moviebot-squiezr3n3t55xzc3c46awndia.us-east-1.es.amazonaws.com/directors-v2/director";
```

Comment:
in the director query we seem to be using `director-v6` but in the worker we seem to be using `director-v2` possible using the wrong index ?